dascore/core/patch.py

```diff
@@ -139,12 +142,19 @@ def attrs(self) -> FrozenDict:
     # --- processing funcs
 
     select = dascore.proc.select
+    sel = dascore.proc.sel
+    isel = dascore.proc.isel
     decimate = dascore.proc.decimate
     detrend = dascore.proc.detrend
     pass_filter = dascore.proc.pass_filter
     aggregate = dascore.proc.aggregate
     abs = dascore.proc.abs
 
+    # --- io funcs
+
+    save_pickle = dascore.utils.save_pickle
+    load_pickle = dascore.utils.load_pickle
+
```

Comment:
We should create a new io module for pickle. However, I was just thinking about refactoring this so we can leave this until we refactor. 

Normally IO would look like this (in ObsPy)
```python
import dascore as dc
patch = dc.read(picke_path, format='pickle')  # format is optional, it can auto-detect
patch.write("output_path.pkl", "pickle")
```